packages/cdk/lib/stack-input.ts

```diff
@@ -26,6 +26,7 @@ export const stackInputSchema = z.object({
       videoAnalyzer: z.boolean().optional(),
       diagram: z.boolean().optional(),
       speechToSpeech: z.boolean().optional(),
```

Comment:
Thank you for the corrections. The speechToSpeech option is unnecessary, so could you remove it in this PR?

Reply:
I've updated the PR and removed the speechToSpeech option as requested.